doc/guides/porting/board_porting.rst

```diff
@@ -120,6 +120,14 @@ Board
 A board implements an SoC with all its features, together with peripherals
 available on the board that differentiates the board with additional interfaces
 and features not available in the SoC.
+A board port on Zephyr is typically made of two parts:
+
+- A hardware description (usually done by device tree), which specifies embedded
+  SoC reference, connectors and any other hardware components sch as LEDs,
+  buttons, sensors or communication peripheral (USB, BLE controller, ..)
+
+- A software configuration (done using Kconfig) which configures and optionally
+  enables software features that are needed to operate the board.
```

Comment:
Doesn't the same apply for device tree? I.e. we need to configure and enable things needed to properly operate the board?  E.g. uart pins, enabling peripherals etc.

Reply:
Agree it is not clear. Can you check the proposal made to Anas below ?